cart-page/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>cart-page</title>
+    <link rel="stylesheet" href="https://bitui.netlify.app/common-css/main.css">
+    <link rel="stylesheet" href="/common-css/main.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="navbar-container">
```

Comment:
All the items in the nav is supposed to navigate somewhere so you may need to add the links to them.